docs/content/latest/explore/_index.md

```diff
@@ -181,10 +183,10 @@ Use [yb-ctl](../admin/yb-ctl) local cluster management utility to explore the co
     </a>
   </div>
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="backup-restore/point-in-time-recovery-ysql/">
+    <a class="section-link icon-offset" href="cluster-management/point-in-time-recovery-ysql/">
       <div class="head">
         <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
```

Comment:
Icon used for Manage is section_icons/quick_start/sample_apps.png

Reply:
I changed the icon (I like the new one better here)